applications/openshift/registry/ocp_allowed_registries/rule.yml

```diff
@@ -8,6 +8,15 @@ description: |-
     and pods. This configuration setting ensures that all registries other than
     those specified are blocked.
 
+    You can set the allowed repositories by applying the following manifest using
+    <code>oc patch</code>:
+    <code>
+    spec:
+      registrySources:
+        allowedRegistries:
+        - my-trusted-registry.internal.example.com
+    </code>
```

Comment:
I believe that these tags need to be `<pre/>` instead of `<code/>`. I have seen `<code/>` not render multiple lines of code correctly.

Reply:
Pre is now used